doc/source/user_guide/hpc/troubleshooting.rst

```diff
@@ -7,10 +7,41 @@ Troubleshooting
 
 Debugging jobs
 --------------
-- Use ``--output`` and ``--error`` directives in batch scripts to capture
-  standard output and error messages. 
+- Use ``--output`` and ``--error`` directives in batch scripts to captures
+  standard output and error messages to specific files.
+
+  .. code-block:: bash
+
+      #!/bin/bash
+      #SBATCH --job-name=ansys_job            # Job name
+      #SBATCH --partition=qsmall              # Specify the queue/partition name
+      #SBATCH --output=ansys_job.out          # Standard output file
+      #SBATCH --error=ansys_job.err           # Standard error file
+
+      source /home/user/pymapdl/.venv/bin/activate
+      python /home/user/pymapdl.py
 
 - Check SLURM logs for error messages and debugging information.
+- It is also good idea to print the environment variables in your bash script, using 
+  ``printenv``. Additionally, you can filter them using ``grep``.
```

Comment:
Best practice is to follow a code entity with a noun descriptor. I'm not sure in ``printenv`` and ``grep`` are commands or apps. I'll allow you to add the appropriate nouns after these two code entities.

Reply:
Done!